module/VuFind/sql/mysql.sql

```diff
@@ -410,6 +410,46 @@ CREATE TABLE `access_token` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `notifications_broadcasts`
+--
+
+CREATE TABLE `notifications_broadcasts` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `broadcast_id` int(11) DEFAULT NULL,
+  `visibility` tinyint(1) DEFAULT NULL,
+  `visibility_global` tinyint(1) DEFAULT NULL,
+  `priority` int(11) DEFAULT NULL,
+  `author_id` int(11) DEFAULT NULL,
+  `content` text DEFAULT NULL,
+  `color` varchar(255) DEFAULT NULL,
+  `startdate` datetime DEFAULT NULL,
+  `endate` datetime DEFAULT NULL,
```

Comment:
This appears to be the problem -- endate vs. enddate.

Reply:
Sorry about that mistake. The file is fixed.